src/lib/Client.js

```diff
@@ -258,9 +258,9 @@ class KlasaClient extends Discord.Client {
 		this.gateways = new GatewayDriver(this);
 
 		// Register default gateways
-		this.gateways.register('guilds', this.gateways.guildsSchema, this.options.gateways.guilds, false);
-		this.gateways.register('users', undefined, this.options.gateways.users, false);
-		this.gateways.register('clientStorage', this.gateways.clientStorageSchema, this.options.gateways.clientStorage, false);
+		this.gateways.register('guilds', this.gateways.guildsSchema, { download: false, ...this.options.gateways.guilds });
+		this.gateways.register('users', {}, { download: false, ...this.options.gateways.users });
```

Comment:
Should add a gateways.userSchema option and set it here instead of an empty object, even if the option by default is an empty object, keeps it consistent with the other ones